### ISSUE TITLE
Delay in scan start and intermittent flyers

### ISSUE THREAD

**Issue by cmazzoli** [NSLS-II/Bug-Reports#92]
Intermittent flyer (diag6_flyer1):
![image](https://cloud.githubusercontent.com/assets/11068694/13826335/414ca45e-eb8d-11e5-82ae-86e53cbff89b.png)

And again + delay in scan start, as evidenced by topoff_inj happening already with 2 seconds delay after the ct(num=None) was called:

![screenshot from 2016-03-16 15 54 35](https://cloud.githubusercontent.com/assets/11068694/13826836/df91d466-eb8f-11e5-8bf3-f369f45848e2.png)

**Comment by danielballan** [comment]
cc @tacaswell @stuwilkins 

@cmazzoli perceives two problems here:
1. There is significant delay (3-4 seconds) before when the RunStart Document is generated and when the diag6_flyer1 collects its first data point.
2. The flyer seems to make two false starts, collecting two short burst before settling into steady collection.

Both of these effects are reproducible.

(Do not focus on the red line, marking the injection time, which is not related to this issue.)

**Comment by tacaswell** [comment]
Is something else using diag6 as part of the scan?

**Comment by cmazzoli** [comment]
@ dallan
Thanks.
Other thing discovered, if you plot diag6_flyer5 you will something
interesting at the beginning (again reproducible): the false start is
concomitant with the first one of diag6_flyer1 but has just 1 and not 2
delayed zones.
Are they interfering one with the other?

About your comment on the red line: it is referred with the injection
(another flyer, topoff_inj) and it shows that it is actually active, it is
just diag6 which has a delay. Staging?

**Comment by cmazzoli** [comment]
Yes one detector, diag6_monitor but it being in or out our gs.DETS I don't
remember having noticed differences in the delay time at the start of the
scan
